components/push/README.md

```diff
@@ -29,6 +29,9 @@ the application.
 The HTTP endpoint is provided by Mozilla's [autopush service](https://autopush.readthedocs.io/en/latest/),
 and we use the [rust-ece](https://github.com/mozilla/rust-ece) to manage encryption with the cryptographic keys.
 
+Here's a helpful diagram of how the *subscription* flow works at a high level across the moving parts:
+![A Sequence diagram showing how the different parts of push interact](../../docs/diagrams/Push-Component-Subscription-flow.png "Sequence diagram")
```

Comment:
One tiny suggestion for this diagram: it may be helpful to have concrete examples of "Sender" and "Receiving App", just to help the consumer orient to diagram to their own mental model of the world. For example having "Sender (e.g. FxA server)" and "Receiving App (e.g. Fenix)".